app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java

```diff
@@ -516,6 +519,10 @@ protected void initFilterCheckboxes() {
         settings = Utils.getSettingBool(this, Const.CALENDAR_FILTER_CANCELED, true);
         menuItemFilterCanceled.setChecked(settings);
         applyFilterCanceled(settings);
+
+        int savedMin = Integer.parseInt(Utils.getSetting(this, Const.CALENDAR_FILTER_HOUR_LIMIT_MIN, "0"));
+        int savedMax = Integer.parseInt(Utils.getSetting(this, Const.CALENDAR_FILTER_HOUR_LIMIT_MAX, "24"));
```

Comment:
I don't think 0 to 24 are good defaults.
I'd prefer defaulting to something like 8 to 20
Or put some intelligence in it and calculate the hours actually needed to display all lectures...